data/assets/scene/solarsystem/planets/earth/noaa-sos/atmosphere/2000_carbon_monoxide.asset

```diff
@@ -0,0 +1,58 @@
+local globeIdentifier = asset.require("../../earth").Earth.Identifier
+
+
+local Name = "Carbon Monoxide - 2000"
+local Identifier = "noaa-sos-atmosphere-carbon_monoxide"
+local Description = [[ Carbon monoxide is known as "The Silent Killer" because it is a colorless, odorless, tasteless gas that is poisonous to humans and other oxygen breathing organisms. It is also naturally occurring in the atmosphere. Only in high concentrations is carbon monoxide deadly. At the Earth's surface the concentration of carbon monoxide is 100 ppb (parts per billion), however, in urban areas it can get up to 10 ppm (parts per million) or 100 times higher. One of the main anthropogenic sources of carbon monoxide in the atmosphere is emissions from automobiles. In areas with heavy traffic the carbon monoxide can be measured at 50 ppm. The natural sources of carbon monoxide include volcanoes and brush burning. Between the anthropogenic and natural sources, scientists estimate that the annual production of carbon monoxide is 2-5 gigatons.  ]]
+local URL = "https://sos.noaa.gov/catalog/datasets/carbon-monoxide-2000/"
+
+local syncedDirectory = asset.syncedResource({
+	Name = Name,
+	Type = "HttpSynchronization",
+	Identifier = Identifier,
+	Version = 1
+})
+
+local imagesDestination = syncedDirectory .. "images"
+
+local layer = {
+  Identifier = Identifier,
+  Name = Name,
+  Type = "ImageSequenceTileLayer",
+  FolderPath = imagesDestination,
+  Description = Description
+}
+
+local colorbar = {
+  Identifier = Identifier,
+  Name = Name,
+  Type = "ScreenSpaceImageLocal",
+  TexturePath = syncedDirectory .. "colorbar.png",
+  FaceCamera = false,
+  CartesianPosition = { 0.0, -0.5, -2.0 }
+}
+
+asset.onInitialize(function()
+  if not openspace.directoryExists(imagesDestination) then
+    openspace.printInfo("Extracting " .. Name)
+    openspace.unzipFile(syncedDirectory .. "1024.zip", imagesDestination, true)
+  end
+
+  openspace.globebrowsing.addLayer(globeIdentifier, "ColorLayers", layer)
+  openspace.addScreenSpaceRenderable(colorbar);  
+end)
+
+asset.onDeinitialize(function()
+  openspace.globebrowsing.deleteLayer(globeIdentifier, "ColorLayers", layer.Identifier)
+  openspace.removeScreenSpaceRenderable(colorbar.Identifier)
+end)
+
+
+asset.meta = {
```

Comment:
For some reason, the meta info for the added NOAA layers are not shown in the GUI. Should be investigated

Reply:
The `asset.export` was missing. So the meta information was missing.  The description is there but layers in general don't show the meta information it seems